docs/docs/components/lifecycle-operator/keptn-apps.md

```diff
@@ -123,3 +125,30 @@ annotation/label is populated:
   (in other words, neither the `keptn.sh/app`
   or `app.kubernetes.io/part-of` annotation/label is populated),
   one `KeptnApp` resource is created automatically for each workload.
+
+## Keptn non-blocking deployment functionality
+
+Keptn provides an option to disable the default deployment blocking functionality
+when pre-deployment tasks or evaluations (on KeptnApp or KeptnWorkload level) fail.
+By creating a [KeptnConfig](../../reference/crd-reference/config.md) resources and
+setting the `.spec.blockDeployment` parameter of to `false` the blocking
+behavior for Keptn is disabled and therefore all applications will get deployed
+to the cluster whether the pre-deployment tasks or evaluations fail.
+
+This behavior is valuable if you want to execute a dry-run of the
+tasks/evaluations for the application, but still have your application deployed
+to the cluster not depending on the results of the pre-checks.
+
+If the checks of the application fail, the state of the deployment phase
+are marked as `Warning` and you are able to inspect which
```

Comment:
I think somewhere we have the different possible states documented, we should probably add the warning state as well...

Reply:
Searched for it, but I do not see that anywhere, can you point me to it?